src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -13,5 +13,9 @@ public static string GetUkprn(this ClaimsPrincipal user)
             return user.FindFirst(ProviderClaims.ProviderUkprn)?.Value;
         }
 
+        public static string GetEmailAddress(this ClaimsPrincipal user)
+        {
+            return user.FindFirstValue(ProviderClaims.Email) ?? user.FindFirstValue("email");
```

Comment:
Do you understand this - I don't - just checking you are doing this ?? for a good reason

Reply:
So I was working on the Admin stuff there were some claims that existed with more than one name.  so when i 'copied' the code snipped from Recruit I though this was one of those cases.  however, having debugged the code and checked my claims I can only see one email entry with the key "email" .  I think it has been "email" since DFESignIn was added, so I have Changed the key to public static string Email => "email"; and this will also solve the sonarcloud issue